src/dataLoading/dataHuBMAP.ts

```diff
@@ -27,8 +27,9 @@ export function loadHuBMAPData(uuids: string[], ordering?: dataOrdering) {
       if (values[0] && values[1]) {
         const obsSetsList = values[0][0];
         const filtering = values[0][1];
-        const hubmapIDs = values[1][0];
-        const hubmapIDsFiltered = hubmapIDs.filter((_, index) => filtering[index] === 1);
+        const uuidsFiltered = uuids.filter((_, index) => filtering[index] === 1);
+        const uuid_to_hubmap_id = values[1][0];
```

Comment:
Since camelCase is the usual standard for JS/TS variable names (when possible, i.e. when it's not an API response field), this is my only suggestion:
```suggestion
        const uuidToHubmapId = values[1][0];
```